www/class/config-generate-remote/generate.class.php

```diff
@@ -0,0 +1,304 @@
+$configFile = realpath(__DIR__ . "/../../../config/centreon.config.php");
+if ($configFile !== false) {
+    require_once $configFile;
+}
+
+require_once __DIR__ . '/backend.class.php';
+require_once __DIR__ . '/abstract/object.class.php';
+require_once __DIR__ . '/hosttemplate.class.php';
+require_once __DIR__ . '/command.class.php';
+require_once __DIR__ . '/timeperiod.class.php';
+require_once __DIR__ . '/hostgroup.class.php';
+require_once __DIR__ . '/servicegroup.class.php';
+require_once __DIR__ . '/contact.class.php';
+require_once __DIR__ . '/contactgroup.class.php';
+require_once __DIR__ . '/servicetemplate.class.php';
+require_once __DIR__ . '/service.class.php';
+require_once __DIR__ . '/media.class.php';
+require_once __DIR__ . '/macroService.class.php';
+require_once __DIR__ . '/host.class.php';
+require_once __DIR__ . '/serviceCategory.class.php';
+require_once __DIR__ . '/resource.class.php';
+require_once __DIR__ . '/engine.class.php';
+require_once __DIR__ . '/graph.class.php';
+require_once __DIR__ . '/manifest.class.php';
+require_once __DIR__ . '/hostCategory.class.php';
+require_once __DIR__ . '/curves.class.php';
+require_once __DIR__ . '/trap.class.php';
+require_once __DIR__ . '/relations/viewImgDirRelation.class.php';
+require_once __DIR__ . '/relations/viewImageDir.class.php';
+require_once __DIR__ . '/relations/extendedServiceInformation.class.php';
+require_once __DIR__ . '/relations/extendedHostInformation.class.php';
+require_once __DIR__ . '/relations/hostServiceRelation.class.php';
+require_once __DIR__ . '/relations/hostTemplateRelation.class.php';
+require_once __DIR__ . '/relations/macroHost.class.php';
+require_once __DIR__ . '/relations/timeperiodExceptions.class.php';
+require_once __DIR__ . '/relations/contactgroupHostRelation.class.php';
+require_once __DIR__ . '/relations/contactgroupServiceRelation.class.php';
+require_once __DIR__ . '/relations/contactHostRelation.class.php';
+require_once __DIR__ . '/relations/contactServiceRelation.class.php';
+require_once __DIR__ . '/relations/contactHostcommandsRelation.class.php';
+require_once __DIR__ . '/relations/contactServicecommandsRelation.class.php';
+require_once __DIR__ . '/relations/hostcategoriesRelation.class.php';
+require_once __DIR__ . '/relations/serviceCategoriesRelation.class.php';
+require_once __DIR__ . '/relations/hostgroupRelation.class.php';
+require_once __DIR__ . '/relations/servicegroupRelation.class.php';
+require_once __DIR__ . '/relations/trapsServiceRelation.class.php';
+require_once __DIR__ . '/relations/trapsVendor.class.php';
+require_once __DIR__ . '/relations/trapsGroupRelation.class.php';
+require_once __DIR__ . '/relations/trapsGroup.class.php';
+require_once __DIR__ . '/relations/trapsMatching.class.php';
+require_once __DIR__ . '/relations/trapsPreexec.class.php';
+require_once __DIR__ . '/relations/nagiosServer.class.php';
+require_once __DIR__ . '/relations/cfgResourceInstanceRelation.class.php';
+
+class Generate
```

Comment:
CamelCase only for attribute names and variables.
Please to add the PHP documentation for all methods.